hooks/pre/35-change-evr-extension-ownership.rb

```diff
@@ -0,0 +1,17 @@
+if local_postgresql?
```

Comment:
I'd also limit it to when `postgresql-evr` is installed
```suggestion
if local_postgresql? && execute("rpm -q postgresql-evr", false, false)
```

We already have that code in 2 other places so perhaps it's also time for a common wrapper method like `package_installed?` `oauth_aio_gem_installed?` already has most of that logic anyway

Reply:
Updated with suggested code..Didn't mess with wrapper right now..